tests/conftest.py

```diff
@@ -302,7 +302,7 @@ def gensalt_mock(rounds=12, prefix=b"2b"):
 
 
 @pytest.fixture
-def hass_storage():
+def hass_storage() -> Generator[dict[str, Any], None, None]:
```

Comment:
Could you please also adjust the pylint plugin accordingly?

Reply:
done